ui/packages/catalog-ui-search/src/main/webapp/component/details-interactions/details-interactions.view.js

```diff
@@ -42,6 +43,13 @@ module.exports = Marionette.LayoutView.extend({
     this.$el.toggleClass('is-summary', this.options.summary)
   },
   handleTypes() {
+    const canEdit = this.options.selectionInterface
+      .getSelectedResults()
+      .reduce((canEdit, result) => {
```

Comment:
This would probably be more clear as
`.every(result => user.canWrite(result.get('metacard').get('properties'))`
as it's much more declarative than the reduce.